cmd/frontend/graphqlbackend/git_commits.go

```diff
@@ -88,8 +88,13 @@ func (r *gitCommitConnectionResolver) compute(ctx context.Context) ([]*gitdomain
 			return nil, errors.Wrap(err, "failed to resolve revision range")
 		}
 
+		revisionRange := r.revisionRange
+		if revisionRange == "" {
+			revisionRange = "HEAD"
+		}
+
```

Comment:
Hey maybe we can use that new function here:

```suggestion
		revisionRange := cmp.Or(r.revisionRange, "HEAD")
```

Reply:
oh nice